NobleJoseph580/Chaganaserry

```diff
@@ -0,0 +1,3 @@
+Chaganaserry 
+9.454790N,76.551642E
+https://user-images.githubusercontent.com/116492843/198869832-c1c2cc98-a482-4a4b-9ff0-06e2636cab7a.jpg
```

Comment:
Please provide more precise values. Changanacherry is a locality, not a pinpoint location. I hope the image is of a Church. Please update the contribution using the name of the Church.